### ISSUE TITLE
HTML output

### ISSUE THREAD

**Issue by alexblack** [NotionX/react-notion-x#82]
Hi, is there any way to use standard header tags like <h1/> and <h2/> when rendering? Using headers is a good practice for a few reasons including SEO.

Eg `<h1 class="notion-h-title">My title</h1>` instead of `<span class="notion-title">My title</span>`

**Comment by normdoow** [comment]
Yeah, I think we need to make this change.

**Comment by transitive-bullshit** [comment]
Agreed; this would be a great change. Just need to make sure that the `h` tag styles are properly reset so they don't render differently across browsers.

**Comment by pbteja1998** [comment]
@normdoow @transitive-bullshit Can I work on this?

**Comment by normdoow** [comment]
@pbteja1998, sorry I didn't see your comment here. That would have been a great one for you to work on.
I just made the changes here: https://github.com/NotionX/react-notion-x/pull/109

**Comment by pbteja1998** [comment]
Wow. Thanks again 😀
